.github/workflows/docker-image.yml

```diff
@@ -51,8 +51,8 @@ jobs:
         uses: docker/metadata-action@369eb591f429131d6889c46b94e711f089e6ca96 # v5
         with:
           images: |
-            flexget/flexget
-            ghcr.io/flexget/flexget
+            ${{ secrets.DOCKERHUB_USERNAME }}/flexget
```

Comment:
I just realized this isn't quite right, and is why we haven't had a release on docker hub in a few days. This should be 'flexget' but my docker username is 'gazpachoking'

Reply:
I'm sorry for causing such an error. I've submitted a PR #4160 for this.